specs/searchHomePage.spec.js

```diff
@@ -0,0 +1,13 @@
+let homePage = require('../pageObject/homePage');
+let detailsPage = require('../pageObject/detailsPage');
+
+describe('Search on home page', () => {
+    it('Positive', async () => {
+        await homePage.get();
+        await homePage.searchTOpFor('dress');
+        expect(await homePage.searchResults()).toEqual('0 results have been found.');
```

Comment:
What are we checking with this expect? (there should be 0 results?)
The task is to parse number from error string and check is it is **> 0**